src/blob.jl

```diff
@@ -127,6 +153,15 @@ end
         end
     end
 end
+# TODO: This generates >2x llvm instructions, even though it _is_ type stable...
+# Need to see if there's something better we can do here.
```

Comment:
Out of curiosity, what do the additional instructions look like?

Reply:
sent on slack

Reply:
Oh, actually, i can upload them here too
[Archive.zip](https://github.com/RelationalAI-oss/Blobs.jl/files/15254910/Archive.zip)